tools/submission/submission_checker.py

```diff
@@ -1785,6 +1785,8 @@ def check_extra_files(path, target_files):
                 if target_file not in files:
                     check_pass = False
                     missing_files.append(f"{os.path.join(path, dir, target_file)}.png")
+            if "captions" not in files:
```

Comment:
Should this be "captions.txt" ?

Reply:
I am currently ignoring the file extensions and removing them from the filenames. So that you can send a `.png` or `.jpg`. So the files array would only contain `.../captions`